indico/web/forms/widgets.py

```diff
@@ -198,6 +198,42 @@ def __call__(self, field, **kwargs):
                                                      search_url=self.search_url, choices=getattr(field, 'choices', []))
 
 
+class LocationWidget(JinjaWidget):
+    """Renders a collection of fields to represent location"""
+
+    def __init__(self):
+        super(LocationWidget, self).__init__('forms/location_widget.html', single_line=True)
```

Comment:
are you sure you want `single_line` styling (affects the alignment of the label)?